tests/unit/icon.js

```diff
@@ -0,0 +1,89 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	function getContextRequire() {
+		return require.config({
+			context: "icon", // TODO: increment this
```

Comment:
actually I kept it so I can test more things, and I assume it's easier to predict the result of a test if it starts from ground 0.